test/change_health/models/model_test.rb

```diff
@@ -101,6 +101,12 @@ class FakeModel < ChangeHealth::Models::Model
         assert_equal(Date.new(2016, 9, 15), ChangeHealth::Models::PARSE_DATE.call('20160915'))
       end
 
+      it 'handles alternate format' do
```

Comment:
Is it worth adding a date with extra `-` included to ensure it works as expected? (Since you mentioned it in the pr description)

Reply:
I like, yes!